auth/api/grpc/responses.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package grpc
+
+type identityRes struct {
+	id    string
+	email string
+	err   error
+}
+
+type issueRes struct {
+	value string
+	err   error
+}
+
+type authorizeRes struct {
+	authorized bool
+	err        string
+}
+type membersRes struct {
```

Comment:
Should this contain an err field, as well? Please put an empty line between type definitions (`membersRes` and `emptyRes`).